components/inputs/test/input-time.visual-diff.js

```diff
@@ -10,7 +10,7 @@ describe('d2l-input-time', () => {
 
 	before(async() => {
 		browser = await puppeteer.launch();
-		page = await visualDiff.createPage(browser, { viewport: { width: 300, height: 800 } });
+		page = await visualDiff.createPage(browser, { viewport: { width: 650, height: 800 } });
```

Comment:
Needed to change the width of the viewport to 650px > 616px to get the normal styling. This caused the text in the time-picker footer to expand. Those are the only visual diff changes in the time-picker screenshots.